src/port_forwarding.rs

```diff
@@ -36,6 +36,8 @@ impl From<ForwardType> for native_mux_impl::ForwardType {
 #[derive(Clone, Debug, Eq, PartialEq, Hash)]
 pub enum Socket<'a> {
     /// Unix socket.
+    #[cfg(not(windows))]
```

Comment:
It feels like UnixSocket should be guarded by `#[cfg(unix)]`, not by `#[cfg(not(windows))]`.

Reply:
Done.